app/uk/gov/hmrc/hmrcemailrenderer/templates/voa/cca/exisiting/ccaEnrolmentConfirmationExisiting.scala.txt

```diff
@@ -0,0 +1,16 @@
+@(params: Map[String, Any])
+
+<Valuation Office Agency – check and challenge registration>
+
+Dear <name>,
```

Comment:
You have to call `@params("name")` or the email will never get rendered and will throw errors